packages/terra-form-input/docs/README.md

```diff
@@ -0,0 +1,32 @@
+# Terra Form Input
+
+Generic input which represents an HTML input element directly.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-form-input`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Input from 'terra-form-input';
+
+<Input
+  name="description"
+  value="I'm a software engineer working at Cerner"
+  required
+/>```
```

Comment:
Previewing this markdown file shows a couple syntax issues

Reply:
The closing tick marks (```) should be moved to a new line

<pre><code>
/>
```
</code></pre>

Reply:
Good catch. Fixed 629fc064